guides/source/configuring.md

```diff
@@ -473,6 +473,8 @@ in controllers and views. This defaults to `false`.
 
 * `config.active_record.queues.destroy` allows specifying the Active Job queue to use for destroy jobs. When this option is `nil`, purge jobs are sent to the default Active Job queue (see `config.active_job.default_queue_name`). It defaults to `nil`.
 
+* `config.active_record.enumerate_columns_in_select_statements` when truthy, will always include column names in `SELECT` statements, and avoid wildcard `SELECT * FROM ...` queries. This avoids prepared statement cache errors when adding columns to a Postgres database. Defaults to `false`.
```

Comment:
Just curious, but why does it default to `false`? Is there any drawback to enumerating the columns instead of using `*`?

Reply:
As far as I know there's no drawback, but it's off by default to guarantee backwards compatibility by maintaining existing behavior.

Reply:
One small drawback is that the query logs will be bigger/more verbose when showing the SQL generated by ActiveRecord.